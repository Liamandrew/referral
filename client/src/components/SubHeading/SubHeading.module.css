.subHeading {
    color: #868383;
    text-transform: uppercase;
    margin-block-end: unset;
    margin-bottom: 16px;
}

.underline {
    border-bottom: 1px solid #cdcdcd;
}
