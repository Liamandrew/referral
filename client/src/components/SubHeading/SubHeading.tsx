import React from "react";
import style from "./SubHeading.module.css";

const SubHeading: React.FC = ({ children }) => (
    <div className={style.underline}>
        <h4 className={style.subHeading}>{children}</h4>
    </div>
);

export { SubHeading };
