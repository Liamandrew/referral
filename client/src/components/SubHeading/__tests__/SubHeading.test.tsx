import React from "react";
import { render } from "@testing-library/react";
import { SubHeading } from "../SubHeading";

describe("<SubHeading>", () => {
    test("should match snapshot", () => {
        const { container } = render(<SubHeading>text</SubHeading>);

        expect(container).toMatchSnapshot();
    });
});
