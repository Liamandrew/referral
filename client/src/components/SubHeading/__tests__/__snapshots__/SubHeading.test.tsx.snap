// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SubHeading> should match snapshot 1`] = `
<div>
  <div
    class="underline"
  >
    <h4
      class="subHeading"
    >
      text
    </h4>
  </div>
</div>
`;
